packages/devtools_app/lib/src/screens/memory/panes/allocation_tracing/allocation_profile_class_table.dart

```diff
@@ -118,29 +118,45 @@ class _AllocationTracingTableState extends State<AllocationTracingTable> {
 
   @override
   Widget build(BuildContext context) {
```

Comment:
this looks broken

Reply:
How so? Seems to work fine on my end.

Reply:
weird. maybe git diff is struggling:
![Screen Shot 2022-08-19 at 12 39 45 PM](https://user-images.githubusercontent.com/43759233/185695007-dbc3b4d5-b951-4931-be19-7f4e702b9cd8.png)